content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -356,6 +357,20 @@ prisma init --datasource-provider sqlite
 
 The command output contains helpful information on how to use the generated files and begin using Prisma with your project.
 
+#### Run `prisma init --preview-feature`
+
+```terminal
+prisma init --preview-feature multiSchema
+```
+
+The command sets `multiSchema` as a Preview feature in your schema.
+
+```terminal
+prisma init --preview-feature multiSchema --preview-feature metrics
+```
+
+The command sets `multiSchema` and `metrics` as the Preview features in your Prisma schema.
```

Comment:
Should this maybe show the resulting `previewFeatures`?

Reply:
Done ✅

I've used the CodeWithResult component here.